src/openms/source/TRANSFORMATIONS/FEATUREFINDER/MultiplexFiltering.cpp

```diff
@@ -76,27 +99,32 @@ namespace OpenMS
     }
     
     // blacklist low-intensity peaks
-    for (MSExperiment::ConstIterator it_rt = exp_picked_.begin(); it_rt < exp_picked_.end(); ++it_rt)
+    for (MSExperiment::ConstIterator it_rt = exp_centroided_.begin(); it_rt < exp_centroided_.end(); ++it_rt)
     {
       for (MSSpectrum::ConstIterator it_mz = it_rt->begin(); it_mz < it_rt->end(); ++it_mz)
       {
         if (it_mz->getIntensity() < intensity_cutoff_)
         {
-          blacklist_[it_rt - exp_picked_.begin()][it_mz - it_rt->begin()] = 666;
+          blacklist_[it_rt - exp_centroided_.begin()][it_mz - it_rt->begin()] = 666;
         }
       }
     }
     
   }
+  
+  MSExperiment MultiplexFiltering::getCentroidedExperiment()
+  {
+    return(exp_centroided_);
```

Comment:
we usually don't use () around return value

Reply:
consider returning a const reference here

Reply:
> consider returning a const reference here

yes, unless you want to make a copy I also suggest a const ref